.github/workflows/update_directory.yml

```diff
@@ -18,7 +18,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
-      - uses: actions/setup-python@master
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
I used the latest stable version of python: https://github.com/actions/python-versions/releases